### DISCUSSION TITLE
If I don’t get the correct text on the image from the prompt I provide, what is the solution for this? Please provide it."

### DISCUSSION THREAD

**Discussion by jenya12369** [lllyasviel/Fooocus#3630] [Q&A]
![image](https://github.com/user-attachments/assets/6ab877a1-396f-41cb-9ab4-a7a40f832397)

**Comment by kalle07** [comment]
please what ?

**Reply by jenya12369** [reply]
"The image I generate doesn’t display the complete text on it. Is there any solution for this?"

**Reply by kalle07** [reply]
EXPLANE !!!
which model, which prompt, what text ?
you are stoned ? ^^

**Reply by jenya12369** [reply]
Modal = Fooocus Modal 
 
Prompt = An innovative and futuristic image of Roku working in harmony with DIRECTV Stream. The sleek Roku device is shown connected to a large, flat-screen TV. The DIRECTV Stream logo is visible on the screen, displaying a variety of popular channels. The room has a modern, open concept design, with a comfortable couch and a cozy ambiance. The overall tone of the image is innovative and user-friendly, reflecting the seamless integration of technologies
  
 Text = Image in text is not displaying Perfectly that needs to correct 
 
 And I am sober  hehe
 
 == The solution I want is for the text in the image to display properly. I need assistance to ensure the image is displayed correctly.
